app/src/main/java/com/jerboa/db/dao/AccountDao.kt

```diff
@@ -0,0 +1,34 @@
+package com.jerboa.db.dao
+
+import androidx.lifecycle.LiveData
+import androidx.room.Dao
+import androidx.room.Delete
+import androidx.room.Insert
+import androidx.room.OnConflictStrategy
+import androidx.room.Query
+import androidx.room.Update
+import com.jerboa.db.entity.Account
+
+@Dao
+interface AccountDao {
+    @Query("SELECT * FROM account")
+    fun getAll(): LiveData<List<Account>>
+
+    @Query("SELECT * FROM account where current = 1 limit 1")
```

Comment:
Made a query to get the current account instead of getting all and fetching the first here https://github.com/dessalines/jerboa/pull/973/files#diff-697367ddff23d571ded643b597f312aeba4b29669b058e9edb03edf9bb030d92L24